testing/src/main/java/org/motechproject/nms/testing/service/impl/TestingServiceImpl.java

```diff
@@ -217,7 +217,7 @@ private void truncateTable(final String table) {
 
             @Override
             public String getSqlQuery() {
-                return String.format("DELETE FROM %s WHERE 1=1", table);
```

Comment:
Interesting, I added this to avoid getting sql "can't delete entire table" errors, you're not getting this?